src/view/patching_mode/patching_selector.py

```diff
@@ -0,0 +1,113 @@
+# coding=utf-8
+"""selector for Patching witch holds all Patching Universes"""
+import random
+import re
+from typing import TYPE_CHECKING
+
+from PySide6 import QtWidgets, QtGui
+
+import proto
+from model.broadcaster import Broadcaster
+from model.patching_universe import PatchingUniverse
+from ofl.patching_dialog import PatchingDialog
+from view.patching_mode.patch_plan_widget import PatchPlanWidget
+
+if TYPE_CHECKING:
+    from view.main_window import MainWindow
+
+
+class PatchingSelector(QtWidgets.QTabWidget):
+    """selector for Patching witch holds all Patching Universes"""
+
+    def __init__(self, broadcaster: Broadcaster, parent: "MainWindow"):
+        super().__init__(parent=parent)
+        self._broadcaster = broadcaster
+        self._broadcaster.connection_state_updated.connect(self._connection_changed)
+        self._broadcaster.add_universe.connect(self._add_universe)
+        self._patch_planes: list[PatchPlanWidget] = []
+        self.setTabPosition(QtWidgets.QTabWidget.TabPosition.West)
+        self.addTab(QtWidgets.QWidget(), "+")
+        # self.currentChanged.connect(self._tab_changed)
+        self.tabBarClicked.connect(self._tab_clicked)
+        self.tabBar().setCurrentIndex(0)
+
+        self._toolbar: list[QtGui.QAction] = []
+        patch_button = QtGui.QAction("Patch")
+        patch_button.triggered.connect(self.patch)
+        self._toolbar.append(patch_button)
+
+    @property
+    def toolbar(self) -> list[QtGui.QAction]:
+        """toolbar for patching_mode"""
+        return self._toolbar
+
+    def _generate_universe(self) -> None:
+        """add a new Universe to universe Selector"""
+        # TODO add universe Dialog
+        universe = PatchingUniverse(proto.UniverseControl_pb2.Universe(
```

Comment:
This will do until coal is over but we should definitely introduce a proper dialogue for this. :-)